pages/dev/send-tokens/interchain-tokens.mdx

```diff
@@ -0,0 +1,58 @@
+import { Callout } from 'nextra-theme-docs'
+
+# Interchain Tokens
+
+Axelar has [several ways](overview) to send [supported tokens](/resources/testnet#assets), but what if you want to send another ERC20 and have it be available on multiple chains? Axelar can help with our new Interchain Tokens!
+
+Our Interchain Token tools register an existing ERC20 (we'll help you create a new one one if you don't have one) and make it available on multiple chains.
+
+<Callout type="warning" emoji="⚠️">
+ Interchain Tokens are currently only available on testnets. Stay tuned to be among the first to find out about our mainnet launch.
+</Callout>
+
+## Creating an Interchain Token
+The first step is to visit the [https://testnet.services.axelar.dev/](Interchain Token Portal). Select the correct source network in your wallet, and then enter the address of an existing ERC20 token, or choose to create a new one. 
+
+If you create a new one, you can choose an initial supply or mint new tokens later.
+
+It's also simple to [use Remix to create your own ERC-20 for testing](https://remix.ethereum.org/axelarnetwork/axelar-docs/blob/stephen/interchain-token-docs/public/samples/interchain-token-simple.sol).
+
+[Interchain Token Portal Testnet](https://testnet.services.axelar.dev/) (Mainnet coming soon)
+
+Follow the prompts to verify the token address or create your token, and then choose the chains you'd like to deploy your interchain token to. Axelar will configure the token linker on the destination chains, and make your ERC20 token available on all destination chains.
+
+Take note in the upper right hand corner of your Token ID, you'll need this when moving your token between blockchains.
+
+## Interacting with your Interchain Token
+You can interact with your original token on the source chain and the wrapped tokens on all other chains identically with any other ERC-20 token. You can send, transfer, approve, and use all of the methods you are used to.
+
+## Transferring Tokens Between Chains using Axelar
```

Comment:
will the erc-20 that we register on the source be "wrapped" right away to expose the cross-chain call, or we still need to do work for it? @re1ro

Reply:
@sergeynog as right now we still need to implemented this functionality as we don't have a cross-chain token ready yet. I think we should make this automatic in the future and deploy source chain wrapped version as we register the origin token, so every registered token will always have a wrapped version. We should also probably decide on the token symbol prefixing standard for this.